README.md

```diff
@@ -21,10 +21,40 @@ To build the image from the Dockerfile, run
 sudo docker build -t analythium/shinyproxy-demo .
 ```
 
-Test locally
+
+
+## Develop and test locally
+```bash
+docker-compose up --build
+```
+then visit `http://localhost:4000`.  To develop, simply make changes to your Shiny app and then reload the page!
+
+
+
+
+## Production build, test, and deploy
+Follow ALL of the steps below to deploy the latest version of your app...
+
+To build the image from the Dockerfile, run
 ```bash
-docker run -p 4000:3838 analythium/shinyproxy-demo
+sudo docker build -f prod.Dockerfile -t yourregistryusername/shinyproxy-test:prod .
 ```
-then visit `127.0.0.1:4000`.
+
+then run locally with
+```bash
+docker run -p 4000:3838 yourregistryusername/shinyproxy-test:prod
```

Comment:
this should say `shinyproxy-demo`.  feel free to fix these if you approve everything else.  sorry about that.